infra/base-images/base-runner/Dockerfile

```diff
@@ -82,6 +82,9 @@ RUN wget https://repo1.maven.org/maven2/org/jacoco/org.jacoco.cli/0.8.7/org.jaco
     echo "37df187b76888101ecd745282e9cd1ad4ea508d6  /opt/jacoco-agent.jar" | shasum --check && \
     echo "c1814e7bba5fd8786224b09b43c84fd6156db690  /opt/jacoco-cli.jar" | shasum --check
 
+COPY install_javascript.sh /
```

Comment:
Yeah this adds 60 MB (about 30% to the current 220MB size)

Reply:
@jonathanmetzman Is this a big problem? I don't see any easy way around it.

Reply:
Why do we need it if the fuzzers will bundle node?

Reply:
I guess i'm already installing node in cifuzz-base (which inherits from base-runner) so maybe I should update cifuzz myself after this lands.

Reply:
We do not bundle node with the fuzzers. We install all dependencies of the JS project during the build step and call the Jazzer.js CLI directly so that we don't have to install npm